examples/Stochastic Simulation

```diff
@@ -0,0 +1,170 @@
+Importa library
+
+import os
+import pandas as pd
+import flopy
+import numpy as np
+import matplotlib.pyplot as plt
+import flopy.utils.binaryfile as bf
+from ipywidgets import interact
+import shutil
+import matplotlib.colors
+flopy is installed in C:\Users\Jerson\miniconda3\lib\site-packages\flopy
+Flow Model
+
+Name the model
+
+model_ws='../4_OUT/R_SIM_KRIG'
+modelname='Modelo_Flujo'
+exe_name= '../1_SOLVER/mf2005.exe'
+mf = flopy.modflow.Modflow(modelname, exe_name=exe_name, model_ws=model_ws)
+Spatial discretization
+
+nlay = 30
+nrow = 1
+ncol = 100
+delr=10.
+delc=1.
+top = 510.
+botm = np.ones((nlay), dtype=np.int32)
+for i in range (nlay):
+    botm[i]= 500 -(i*10)
+nper = 3
+nstp = [1]+[1]*2 
+perlen = [1] + [864000]*1 + [2523743999]*1
+steady = [True] + [False]*2
+​
+dis=flopy.modflow.ModflowDis(mf, nlay=nlay, nrow=nrow, ncol=ncol, delr=delr, delc=delc, top=top, 
+                             botm=botm, nper=nper, perlen=perlen, steady=steady, itmuni=1, lenuni=2)
+#botm
+#dis.plot()
+Package BAS
+
+ibound = np.ones((nlay, nrow, ncol), dtype=np.int32)
+ibound[1:,:,0] = -1
+ibound[21:,:,-1] = -1
+#ibound
+Package - strt
+
+bas = flopy.modflow.ModflowBas(mf, ibound=ibound, strt=top)
+#bas.plot()
+Package CHD
+
+chdspd={}
+chdspd[0]=[[i, 0, 0, 500., 500.] for i in range(1,30)]
+for i in range (21,30):
+    chdspd[0].append(([i, 0, 99, 300.,300.])) 
+chdspd[1]=[[i, 0, 0, 500., 500.] for i in range(1,30)]
+for i in range (21,30):
+    chdspd[1].append(([i, 0, 99, 300.,300.])) 
+chdspd[2]=[[i, 0, 0, 500., 500.] for i in range(1,30)]
+for i in range (21,30):
+    chdspd[2].append(([i, 0, 99, 300.,300.])) 
+    
+chd = flopy.modflow.ModflowChd(mf, stress_period_data=chdspd)
+Hydraulic conductivity
+
+kx=pd.read_excel('../3_IN/SIMULACION_K.xlsx', sheet_name=None, header=None) 
+#kx
+hk=kx['KRIGING'].to_numpy()
```

Comment:
Excuse me, is Kriging a module? I'm not sure if it's a function, a module, or a table? I hope someone can help me. I'm a beginner